RecoTracker/IterativeTracking/python/DetachedTripletStep_cff.py

```diff
@@ -252,15 +254,20 @@
 detachedTripletStep = ClassifierMerger.clone()
 detachedTripletStep.inputClassifiers=['detachedTripletStepClassifier1','detachedTripletStepClassifier2']
 
+trackingPhase1.toReplaceWith(detachedTripletStep, detachedTripletStepClassifier1.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorDetachedTripletStep_Phase1'),
+     src = 'detachedTripletStepTracks',
```

Comment:
this is redundant (and was not in the pre-DNN version)

Reply:
Removed redundant line.